src/Spatie/Browsershot/Browsershot.php

```diff
@@ -31,7 +31,11 @@ class Browsershot
     public function __construct($binPath = '', $width = 640, $height = 480, $quality = 60, $timeout = 5000, $backgroundColor = null)
     {
         if ($binPath == '') {
-            $binPath = realpath(dirname(__FILE__).'/../../../bin/phantomjs');
+            if (defined('PHP_WINDOWS_VERSION_BUILD')) {
```

Comment:
Could you add a function to determine the `binPath`. Use early returns to avoid `else`.